src/components/common/query/QueryView.jsx

```diff
@@ -0,0 +1,129 @@
+import React, { Fragment } from 'react';
```

Comment:
Since this is now referred to as the "Data" page for datasets/snapshots, can we rename files to match?